### DISCUSSION TITLE
Get progress bar from wsl --update

### DISCUSSION THREAD

**Discussion by Franklin-Roedel** [microsoft/WSL#11972] [Q&A] [answered]
Can you get the progress bar when running wsl.exe --update? 

When I redirect the output to a file, I am unable to get the progress and only see the following: 

Installing: Windows Subsystem for Linux
Windows Subsystem for Linux has been installed.

Our goal is to pass the progress bar a windows form.

**Comment by OneBlue** [comment, marked as answer]
@Franklin-Roedel: Yes that's by design. Because progress bars need to 'rewrite' the terminal output, they are not showed when redirected to a file (if they did, the output would contain a bunch of terminal codes, which would make the file unusable anyway).
